platform/core/src/services/ColorbarService/ColorbarService.ts

```diff
@@ -0,0 +1,146 @@
+import { PubSubService } from '../_shared/pubSubServiceInterface';
+
+const EVENTS = {
+  COLORBAR_STATE_CHANGED: 'event::ColorbarService:stateChanged',
+};
+
+export default class ColorbarService extends PubSubService {
+  colorbars = {};
+  public static REGISTRATION = {
+    name: 'colorbarService',
+    create: () => {
+      return new ColorbarService();
+    },
+  };
+
+  constructor() {
```

Comment:
which method below is private which public please add.

Reply:
Done